### PR TITLE
Sush repl

### PR DESCRIPTION
**Pull request by AndreTheHunter**
Suppressed repl and logging output

### CODE REVIEW

File: cljmd [hunk 1]
```diff
@@ -26,36 +26,41 @@ run_script(){ # path-to-script.clj & args
...
+			:coordinates m
+			:repositories (merge cemerick.pomegranate.aether/maven-central {"clojars" "https://clojars.org/repo"})))
+
+	(def ^:dynamic *cmdjure-version* "$VERSION")
+	(def ^:dynamic *cwd* (System/getProperty "user.dir"))
+	(def ^:dynamic *script* "$1")
+
+	(binding [*command-line-args* $args]
+		(load-file *script*))
+	nil)
```
Comment (AndreTheHunter) [inline cljmd]:
```clj
(do
   ;stuff
  nil)
```
to shh the repl

File: cljmd [hunk 2]
```diff
@@ -26,36 +26,41 @@ run_script(){ # path-to-script.clj & args
...
 EOF
 
-	DEPS='{:deps {com.cemerick/pomegranate {:mvn/version "1.1.0"}}}'
+	#language=clj
+	DEPS='{:deps          {com.cemerick/pomegranate {:mvn/version "1.1.0" :exclusions [commons-logging]}
```
Comment (AndreTheHunter) [inline cljmd]: Tried to reduce the amount of deps. Others could also probably be excluded, but we need tests first

File: cljmd [hunk 3]
```diff
@@ -26,36 +26,41 @@ run_script(){ # path-to-script.clj & args
...
+                 org.slf4j/slf4j-nop      {:mvn/version "1.7.22"}}
```
Comment (AndreTheHunter) [inline cljmd]: "can't find logger" be gone

File: cljmd [hunk 4]
```diff
@@ -26,36 +26,41 @@ run_script(){ # path-to-script.clj & args
...
+ :override-deps {org.slf4j/slf4j-api      {:mvn/version "1.7.22"}
+                 org.slf4j/jcl-over-slf4j {:mvn/version "1.7.22"}}}'
```
Comment (AndreTheHunter) [inline cljmd]: Resolve conflicts
Obviously pomegranate hasn't run `lein deps :tree`

Comment (AndreTheHunter) [reply cljmd]: Hmm looks like this is not working...

**Review by axrs — APPROVED** [review]
Thanks :)

**Pull request merged**